src/libs/masq.js

```diff
@@ -1 +1,112 @@
-module.exports = require('masq-client/dist/masq')
+const Masq = require('masq-lib')
+
+const Error = require('../adapters/error').default
+const handleError = (fct, msg, e) => {
+  Error.sendOnce('masq_store', fct, msg, e)
+}
+
+function MasqStore(config) {
+  this.masq = null
+  this.loginLink = null
+  this.config = config
+}
+
+MasqStore.prototype.getAllPois = async function() {
+  const list = await this.masq.list()
+
+  const filteredKeys = Object.keys(list).reduce((filtered, k) => {
+    if (Poi.isPoiCompliantKey(k)) {
+      filtered.push(k)
+    }
+    return filtered
+  }, [])
+
+  if (filteredKeys.length === 0) {
+    return Promise.resolve([])
+  }
+
+  const valuePromises = filteredKeys.map(k => {
+    return this.masq.get(k)
+  })
+
+  const values = await Promise.all(valuePromises).catch(
+    (e) => {
+      handleError('getAllPois', 'error getting pois', e)
+      throw e
+    })
+
+  return values
+}
+
+MasqStore.prototype.getUserInfo = async function() {
+  const username = await this.masq.getUsername()
+  const profileImage = await this.masq.getProfileImage()
+  return {
+    username,
+    profileImage
+  }
+}
+
+MasqStore.prototype.isRegistered = async function() {
+  return await Promise.resolve(this.masq && this.masq.isLoggedIn())
+}
+
+MasqStore.prototype.registerApp = async function(apps) {
+  // connect to Masq
+  window.open(this.loginLink)
+  await this.masq.logIntoMasq(false)
+}
+
+MasqStore.prototype.unregisterApp = async function() {
+  await this.masq.signout()
```

Comment:
i'm confuse with signout logout & unregister

Reply:
Everything has been renamed to use login/logout